realm/realm-library/src/main/java/io/realm/RealmResults.java

```diff
@@ -153,6 +153,27 @@ public boolean isValid() {
     }
 
     /**
+     * Searches this {@link RealmResults} for the specified object.
```

Comment:
@beeender Do you think checking for `isLoaded` would be correct? If the item is not loaded then return false? What do you think? //cc @cmelchior